packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -143,7 +143,7 @@ function useBlockEditorSettings( settings, postType, postId, renderingMode ) {
 				: undefined;
 
 			function getSectionRootBlock() {
-				if ( renderingMode === 'template-locked' ) {
+				if ( renderingMode.includes( 'template-lock' ) ) {
```

Comment:
What is this change about?

Reply:
Reverted this. I think it was a sideeffect of us trying to fix the tests. But it isn't right